base/applications/rapps/include/winmain.h

```diff
@@ -2,6 +2,10 @@
 
 #include "settings.h"
 
+#define MAINWINDOWCLASSNAME L"ROSAPPMGR2"
+#define MAINWINDOWMUTEX szWindowClass
```

Comment:
Unused?

Reply:
Forgot to replace the old usage, good catch